azurerm/internal/services/apimanagement/resource_arm_api_management_api_operation.go

```diff
@@ -250,15 +250,27 @@ func expandApiManagementOperationRequestContract(input []interface{}) (*apimanag
 	}
 
 	vs := input[0].(map[string]interface{})
+	if vs == nil {
+		return nil, fmt.Errorf("Error mapping input")
+	}
 	description := vs["description"].(string)
 
 	headersRaw := vs["header"].([]interface{})
+	if headersRaw == nil {
+		return nil, fmt.Errorf("Error mapping `header`")
+	}
 	headers := azure.ExpandApiManagementOperationParameterContract(headersRaw)
```

Comment:
So instead could we conditionally assign this here upon missing headers instead of an error?